postgres/cast.go

```diff
@@ -45,6 +45,10 @@ type cast interface {
 	AS_INTERVAL() IntervalExpression
 }
 
+type castArray interface {
+	AS_STRING() jet.ArrayExpression[StringExpression]
```

Comment:
Why separate interface?

Reply:
Was left over cruft from experimentation, removed it